### ISSUE TITLE
Find an alternative to the React Table plugin

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1186]
The only place we're using [React Table](https://react-table.js.org), as far as I can tell, is the Summary Schedule Table. When I was poking around taking inventory of our front-end code, I realized that React Table doesn't render tables -- it renders divs and styles them to look like a table. This is a poor experience for screen readers, because cell contents and headers aren't correlated with each other. (It's also not semantic, but I probably don't have to say that.)

<img width="609" alt="screen shot 2018-11-26 at 6 02 57 pm" src="https://user-images.githubusercontent.com/509309/49115666-a8edca00-f258-11e8-95a0-107adf9066bb.png">

I'd like to figure out what React Table gives us that we think is valuable, and then investigate alternatives. It looks like we're not (currently, anyway) using most of the functionality it offers. The only one I'm aware of is sorting, for which there are a bunch of libraries that will work on our standard tables.

Tagging design in this because dev needs their help in creating the list of criteria for the table replacement.

### This task is done when...
 - [ ] We have a list of what functionality the Summary Schedule table should have
 - [ ] We've found a suitable replacement strategy for React Table
 - [ ] We've implemented that strategy

**Comment by jeromeleecms** [comment]
Will need to replace table that uses this plugin to inherit the existing table structure.
